ros/api/v0/recommendations.py

```diff
@@ -53,7 +53,32 @@ def get(self, host_id):
                 if rule_data:
                     rule_dict = rule_data.__dict__
```

Comment:
@r14chandra - I would prefer to have this logic encapsulated inside class or module. If not possible now then please fill the JIRA for further re-factoring.

Reply:
Will it be okay to have a JIRA for this?

Reply:
+1 for tracking it in a separate JIRA. Only thing, let's not add it as any sub-task for this ticket.

Reply:
created the JIRA -> [RHIROS-436](https://issues.redhat.com/browse/RHIROS-436)